### ISSUE TITLE
jsPDF not working in Chrome

### ISSUE THREAD

**Issue by brettz9** [SVG-Edit/svgedit#254]
jsPDF has not been working in Chrome but it is ok in Firefox (even with #253 to fix #249 ). This is true with the last release, however (at least with current Chrome), so it is not a regression.

Perhaps upgrading the jsPDF version (#51 ) may help with this, though early on, I had only gotten things working with a fork that had been out there...

**Comment by cuixiping** [comment]
On my windows 7 + Chrome 67,  svgedit can export svg successfully.

**Comment by brettz9** [comment]
Export to PDF, you mean?

**Comment by cuixiping** [comment]
PDF yes. I typed wrong just now.

**Comment by brettz9** [comment]
Thanks...Do you have a test file you can paste here? I'd just like to confirm against the exact file...

**Comment by cuixiping** [comment]
I drew some simple shapes on the canvas and exported as pdf. It works well.

Here is the svg code:
```
<svg width="640" height="480.00000000000006" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg">
 <!-- Created with SVG-edit - https://github.com/SVG-Edit/svgedit-->
 <g class="layer">
  <title>Layer 1</title>
  <rect fill="#FF0000" height="109" id="svg_1" stroke="#000000" stroke-dasharray="null" stroke-linecap="null" stroke-linejoin="null" stroke-width="5" width="139" x="20" y="31"/>
  <ellipse cx="526" cy="338.5" fill="#FF0000" id="svg_2" rx="85" ry="120.5" stroke="#000000" stroke-dasharray="null" stroke-linecap="null" stroke-linejoin="null" stroke-width="5"/>
  <circle cx="239" cy="139" fill="#FF0000" id="svg_3" r="64.66065" stroke="#000000" stroke-dasharray="null" stroke-linecap="null" stroke-linejoin="null" stroke-width="5"/>
  <polygon cx="237" cy="275" edge="0" fill="#FF0000" id="svg_4" orient="x" points="237,275 237,275 237,275 237,275 237,275 237,275 " shape="regularPoly" sides="5" stroke="#000000" stroke-width="5" strokeWidth="5" strokecolor="#000000"/>
  <polygon cx="262" cy="250" edge="41.52643" fill="#FF0000" id="svg_5" orient="x" points="404.3244934082031,276.5 352.0323181152344,348.5955810546875 267.4219055175781,321.0575256347656 267.4219055175781,231.94247436523438 352.0323181152344,204.4044189453125 404.3244934082031,276.5 " shape="regularPoly" sides="5" stroke="#000000" stroke-width="5" strokeWidth="5" strokecolor="#000000"/>
 </g>
</svg>
```

**Comment by brettz9** [comment]
Ok, thanks... Very strange, as it's not working for me on the Mac, though I don't see any errors...

**Comment by cuixiping** [comment]
Which version of your Chrome and SVG-Edit ?  You can try again by lastest svgedit of master branch (Download and host to your own machine).

**Comment by brettz9** [comment]
Chrome 67 and master on localhost:8000; testing with `svg-editor.html` or `svg-editor-es.html` (running `npm start`)...

**Comment by cuixiping** [comment]
Was the popup window blocked by your chrome?  Can you see a popup window when export pdf ?

**Comment by brettz9** [comment]
Yes, a new tab appears with "Loading image, please wait..." but it doesn't load...

**Comment by cuixiping** [comment]
What's your OS system and Chrome version?

**Comment by brettz9** [comment]
That'd be awesome, sure! I actually had a couple co-workers (now at Ten Cent) meeting with me as part of a small Open Source group. My family should be moving from Eastern Nanshan to Shekou in September. Feel free to email me at brettz9@yahoo.com or WeChat with ID "brettz9".

**Comment by brettz9** [comment]
Btw, I don't know if this is any better, but we might see whether this might be a better choice than jsPDF: https://github.com/foliojs/pdfkit

**Comment by brettz9** [comment]
Btw, FWIW, I see that there is an SVG kit for pdfkit: https://github.com/devongovett/svgkit

**Comment by cuixiping** [comment]
> Btw, with your PR, I am now seeing export in Chrome, but with your sample from [#254 (comment)](https://github.com/SVG-Edit/svgedit/issues/254#issuecomment-409794710) , we are missing the hexagon... That should probably be filed as its own issue. It's progress though!

I reproduced the issue of missing hexagon. I will debug it later.
